easybuild/easyconfigs/w/Wannier90/Wannier90-2.0.1-intel-2016.02-GCC-4.9.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'MakeCp'
+
+name = 'Wannier90'
+version = '2.0.1'
+
+homepage = 'http://www.wannier.org'
+description = """A tool for obtaining maximally-localised Wannier functions"""
+
+toolchain = {'name': 'intel', 'version': '2016.02-GCC-4.9'}
+toolchainopts = {'usempi': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://www.wannier.org/code']
+
+# ugly hack
+prebuildopts = 'rm make.sys; touch make.sys; '
```

Comment:
@migueldiascosta is `make.sys` a file that gets included in the `Makefile`? There may be no need to remove/empty it at all, since what you specify on the command line via `buildopts` should get preference over what's in the file...